### PR TITLE
Make chat and embedding paths configurable

### PR DESCRIPTION
**Pull request by markpollack**

### CODE REVIEW

**Comment by tzolov** [comment]
@markpollack please update the OpenAI properties documentation: https://docs.spring.io/spring-ai/reference/api/chat/openai-chat.html#_chat_properties

File: models/spring-ai-openai/src/main/java/org/springframework/ai/openai/api/OpenAiApi.java [hunk 1]
```diff
@@ -23,6 +23,7 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonInclude.Include;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import io.micrometer.common.util.StringUtils;
```
Comment (ThomasVitale) [inline models/spring-ai-openai/src/main/java/org/springframework/ai/openai/api/OpenAiApi.java:26]: This should probably be imported from Spring and not Micrometer

`import org.springframework.util.StringUtils;`

Comment (ThomasVitale) [reply models/spring-ai-openai/src/main/java/org/springframework/ai/openai/api/OpenAiApi.java:26]: Actually, it might be an unused import

Comment (markpollack) [reply models/spring-ai-openai/src/main/java/org/springframework/ai/openai/api/OpenAiApi.java:26]: removed

**Comment by markpollack** [comment]
added docs and merged in ee696f33ddc1ef0dd1f4b0953a2178403f589333

**Pull request closed without merging**